.erda/migrations/pipeline/20210908-create-pipeline-triggers.sql

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2021 Terminus, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+CREATE TABLE `pipeline_triggers` (
+  `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT COMMENT '自增id',
+  `event_name` varchar(500) NOT NULL DEFAULT '' COMMENT '触发事件',
+  `pipeline_source` varchar(191) NOT NULL DEFAULT '' COMMENT '来源',
+  `pipeline_yml_name` varchar(191) NOT NULL DEFAULT '' COMMENT '名称',
+  `pipeline_definition_id` bigint(20) NOT NULL DEFAULT '0' COMMENT '定义id',
+  `filter` mediumtext NOT NULL COMMENT '过滤条件',
```

Comment:
TODO.
benchtest for 1000 triggers. Add a screenshot.

Reply:
Running time of 1000 triggers
<img width="264" alt="ssss" src="https://user-images.githubusercontent.com/32703277/132613350-747a4c74-bf8c-42a6-8864-adcadf4e8722.png">

Reply:
todo commit in modules/pipeline/providers/trigger/db/table.go